.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.js]
+indent_style = space
+indent_size = 2
+insert_final_newline = false
```

Comment:
Just checked, our files all have trailing new lines so I think this should be `true` 👍